OpenOversight/app/main/views.py

```diff
@@ -866,173 +867,69 @@ def edit_department(department_id: int):
 @main.route("/department/<int:department_id>")
 def redirect_list_officer(
     department_id: int,
-    page: int = 1,
-    race=None,
-    gender=None,
-    rank=None,
-    min_age=None,
-    max_age=None,
-    last_name=None,
-    first_name=None,
-    badge=None,
-    unique_internal_identifier=None,
-    unit=None,
-    current_job=None,
-    require_photo: Optional[bool] = None,
 ):
     flash(FLASH_MSG_PERMANENT_REDIRECT)
     return redirect(
-        url_for(
-            "main.list_officer",
-            department_id=department_id,
-            page=page,
-            race=race,
-            gender=gender,
-            rank=rank,
-            min_age=min_age,
-            max_age=max_age,
-            last_name=last_name,
-            first_name=first_name,
-            badge=badge,
-            unique_internal_identifier=unique_internal_identifier,
-            unit=unit,
-            current_job=current_job,
-            require_photo=require_photo,
-        ),
+        url_for("main.list_officer", department_id=department_id)
+        + "?"
+        + urlencode(request.args.to_dict(flat=False), doseq=True),
         code=HTTPStatus.PERMANENT_REDIRECT,
     )
 
 
 @main.route("/departments/<int:department_id>")
 def list_officer(
     department_id: int,
-    page: int = 1,
-    race=None,
-    gender=None,
-    rank=None,
-    min_age=None,
-    max_age=None,
-    last_name=None,
-    first_name=None,
-    badge=None,
-    unique_internal_identifier=None,
-    unit=None,
-    current_job=None,
-    require_photo: Optional[bool] = None,
```

Comment:
flask was never setting these vars, so it is basically just a variable initializer list